src/instructions/sgx/enclu.rs

```diff
@@ -0,0 +1,81 @@
+//! ENCLU opcode wrappers
+
+use crate::instructions::sgx::EnclaveError;
+use crate::structures::paging::Page;
+use crate::structures::sgx::*;
+use core::arch::asm;
+
+/// Acknowledge host's request to add a new page or change its permissions.
+pub const EACCEPT: usize = 0x05;
+/// Extend page permissions.
+pub const EMODPE: usize = 0x06;
+/// Acknowledge host's request to add a new page, and populate the page with
+/// the given permissions and data.
+pub const EACCEPTCOPY: usize = 0x07;
+
+/// Acknowledge host's request to add a new page or change its permissions.
+#[inline]
+pub fn accept(secinfo: &SecInfo, dest: Page) -> Result<(), EnclaveError> {
+    let ret;
+    unsafe {
+        asm!(
+            "xchg       {RBX}, rbx",
+            "enclu",
+            "mov        rbx, {RBX}",
+
+            RBX = inout(reg) secinfo => _,
+            in("rax") EACCEPT,
+            in("rcx") dest.start_address().as_u64(),
+            lateout("rax") ret,
+        );
+    }
+
+    match ret {
+        0 => Ok(()),
+        11 => Err(EnclaveError::PageNotTracked),
+        19 => Err(EnclaveError::PageAttributesMismatch),
+        _ => Err(EnclaveError::Unknown),
+    }
+}
+
+/// Extend page permissions.
+#[inline]
+pub fn accept_copy(secinfo: &SecInfo, dest: Page, src: Page) -> Result<(), EnclaveError> {
```

Comment:
You seem to have mixed up the comments for `accept_copy` and `extend_permissions`.